api/docs/code_style.dox

```diff
@@ -161,11 +161,11 @@ strcmp() and memcmp(), the use of ! is counter-intuitive.
 
```

Comment:
> iX: Update a couple small document details.

style: While we use `iX-` as a branch name prefix, we don't put it in the commit title.